packages/@aws-cdk/aws-apigatewayv2-authorizers/README.md

```diff
@@ -0,0 +1,76 @@
+# AWS APIGatewayv2 Authorizers
+<!--BEGIN STABILITY BANNER-->
+
+---
+
+![cdk-constructs: Experimental](https://img.shields.io/badge/cdk--constructs-experimental-important.svg?style=for-the-badge)
+
+> The APIs of higher level constructs in this module are experimental and under active development.
+> They are subject to non-backward compatible changes or removal in any future version. These are
+> not subject to the [Semantic Versioning](https://semver.org/) model and breaking changes will be
+> announced in the release notes. This means that while you may use them, you may need to update
+> your source code when upgrading to a newer version of this package.
+
+---
+
+<!--END STABILITY BANNER-->
+
+## Table of Contents
+
+- [HTTP APIs](#http-apis)
+- [JWT Authorizers](#jwt-authorizers)
+  - [User Pool Authorizer](#user-pool-authorizer)
+
+## HTTP APIs
+
+API Gateway supports multiple mechanisms for controlling and managing access to your HTTP API. They are mainly
+classified into Lambda Authorizers, JWT authorizers and standard AWS IAM roles and policies. More information is
+available at [Controlling and managing access to an HTTP
+API](https://docs.aws.amazon.com/apigateway/latest/developerguide/http-api-access-control.html).
+
+## JWT Authorizers
+
+JWT authorizers allow the use of JSON Web Tokens (JWTs) as part of [OpenID Connect](https://openid.net/specs/openid-connect-core-1_0.html) and [OAuth 2.0](https://oauth.net/2/) frameworks to allow and restrict clients from accessing HTTP APIs.
+
+When configured on a route, the API Gateway service validates the JWTs submitted by the client, and allows or denies access based on its content.
+
```

Comment:
Would be useful to add a brief sentence or two on how clients accessing an API that uses JWT authorizer should interact. You can then just link to an article with more information.

Reply:
Added 🚀